File: Python/circle.py
import math

def main():
    radius = get_radius()
    circumference = calc_circumference(radius)
    area = calc_area(radius)

    print(f"Circumference: {circumference:.2f} cm")
    print(f"Area: {area:.2f} cm²")


def get_radius():
        r = float(input("Enter the radius of the circle (cm): "))
        return r

def calc_circumference(r):
     c = 2 * math.pi * r
     return c

def calc_area(r):
     a = math.pi * pow(r,2)
     return a

if __name__ == "__main__":
    main()

File: C/circle/circle.c
#include <stdio.h>
#include <math.h>

#define PI 3.141592653589793

void calculate_circle(float radius, float *circumference, float *area);

int main(void) {
    float radius, circumference, area;
    
    printf("Radius: ");
    scanf("%f", &radius);

    float *p_circumference = &circumference;
    float *p_area = &area;

    calculate_circle(radius, p_circumference, p_area);

    printf("Circumference: %.2f\n", circumference);
    printf("Area: %.2f\n", area);

    return 0;
}

void calculate_circle(float radius, float *circumference, float *area) {
    *circumference = 2 * PI * radius;
    *area = PI * pow(radius, 2);
}